src/main/java/com/dreammaster/mixin/mixins/early/MixinIntegratedServer.java

```diff
@@ -25,17 +25,10 @@ public MixinIntegratedServer(File workDir, Proxy proxy) {
         super(workDir, proxy);
     }
 
-    @Inject(method = "shareToLAN", at = @At(value = "HEAD"), cancellable = true)
+    @Inject(method = "shareToLAN", at = @At(value = "HEAD"))
     public void dreamcraft$makeShareToLANSingleplayerOnly(WorldSettings.GameType type, boolean allowCheats,
             CallbackInfoReturnable<String> cir) {
-        logger.info(
-                "Share to LAN was initiated. This is not supported by GTNH, no external player will be able to connect");
-
-        this.getConfigurationManager().func_152604_a(type);
-        this.getConfigurationManager().setCommandsAllowedForAll(allowCheats);
-
-        cir.setReturnValue(null);
-        cir.cancel();
+        logger.info("Share to LAN was initiated. This is not officially supported by GTNH");
```

Comment:
This mixin now solely exists to insert this logging call. Let me know if you think this is a waste.